frameworks/C++/lithium/lithium.cc

```diff
@@ -191,7 +190,7 @@ int main(int argc, char* argv[]) {
     std::sort(table.begin(), table.end(),
               [] (const fortune& a, const fortune& b) { return a.message < b.message; });
 
-    char b[100000];
+    char b[2000];
```

Comment:
This isn't acceptable to cover the dynamic sizing rule of the fortunes test and won't pass the new verification step we're working on where we verify dynamic sizing by adding a large number of fortunes during that step

Reply:
Ok. I'll work on a dynamic sizing implementation.

Reply:
@nbrady-techempower This is fixed in my last commit.

Reply:
thanks @matt-42!